README.md

```diff
@@ -110,6 +110,24 @@ Entity body for POST and PUT requests.
 request.body({ firstName: 'Foo', lastName: 'Bar', age: 25 })
 ```
 
+#### .encoding(value)
+
+Set encoding code to the response body.
+
++ `value` (Object) - Represent the enconding code.
+
+**Note:** if value is undefined means this is utf-8 by default.
```

Comment:
replace  with -> **Note:** if value is `undefined` means this is `'utf-8'` by default.